verwijzing en uitleg probeeromgeving (#10)
BRP-API/brp-api.github.io: modules/ROOT/pages/api-proberen.adoc

= Probeer en test de API

> Je kan de API uitproberen en gebruiken bij het ontwikkelen en testen op onze probeeromgeving. De API op deze omgeving is bereikbaar over internet en vereist geen Diginetwerk en je hebt geen PKI-overheid certificaat nodig. 

Reviewer: Feedback op (technische) schrijfvaardigheid:
- Schrijf vanuit de derde persoon (geen jij-vormen)
- Het liefst korte en duidelijke zinnen

```suggestion
De API is bij het ontwikkelen en testen op onze probeeromgeving te gebruiken. Het is bereikbaar over internet. Diginetwerk en PKIoverheid certificaat zijn dus niet nodig. 
```

> Je kan de functionaliteit van de API uitproberen, maar de aansluiting en autorisaties kunnen anders zijn dan in productie. De probeeromgeving is dus bedoeld als eerste stap voor het ontwikkelen.

Reviewer: Feedback op (technische) schrijfvaardigheid:
- Schrijf vanuit de derde persoon (geen jij-vormen)
- Onjuiste tegenstelling weggehaald

```suggestion
De probeeromgeving is toegankelijk gemaakt. Let op dat de aansluiting en autorisaties in productie anders zijn. De probeeromgeving is dus bedoeld als eerste stap voor de ontwikkeling van de aansluiting.
```

= Probeer en test de API

Je kan de API uitproberen en gebruiken bij het ontwikkelen en testen op onze probeeromgeving. De API op deze omgeving is bereikbaar over internet en vereist geen Diginetwerk en je hebt geen PKI-overheid certificaat nodig. 
Je kan de functionaliteit van de API uitproberen, maar de aansluiting en autorisaties kunnen anders zijn dan in productie. De probeeromgeving is dus bedoeld als eerste stap voor het ontwikkelen.

== Toegang vragen

> De API op de probeeromgeving vereist - net als in productie - dat een OAuth token wordt meegestuurd. Hiervoor heb je client credentials nodig. Vraag deze aan door een e-mail te sturen naar mailto:{po-email}[{po-naam}]. Geef in die e-mail aan voor welke organisatie je credentials wilt en geef je contactgegevens op. Die gebruiken we om je te informeren over belangrijke wijzigingen. gepland onderhoud, enz.

Reviewer: Feedback op (technische) schrijfvaardigheid:
- Schrijf vanuit de derde persoon (geen jij-vormen)
- Geen "-" in geschreven tekst

```suggestion
De API op de probeeromgeving gebruikt, net als in productie, de OAuth client credentials flow voor authenticatie. De client credentials kan via email mailto:{po-email}[{po-naam}] worden aangevraagd. Geef in de e-mail aan voor welke organisatie de credentials zijn. Geef ook contactgegevens op van de persoon die benaderd kan worden over eventuele belangrijke wijzigingen, gepland onderhoud, enz.
```
